lua/starfall/libs_sh/enum.lua

```diff
@@ -180,16 +180,26 @@ env.TEXT_ALIGN = {
 env.KEY = {
 	["FIRST"] = 0,
 	["NONE"] = 0,
-	["0"] = 1,
-	["1"] = 2,
-	["2"] = 3,
-	["3"] = 4,
-	["4"] = 5,
-	["5"] = 6,
-	["6"] = 7,
-	["7"] = 8,
-	["8"] = 9,
-	["9"] = 10,
+	["0"] = KEY_0,
```

Comment:
I don't see any reason to switch from 1 to KEY_0 when none of the others do this. Probably nicer to leave them as numbers

Reply:
These numbers do change, see `COUNT`/`KEY_COUNT`?
I couldn't be arsed to figure if some others were changed.
Since when is "nicer" better than "correctness" ?!

Reply:
> Since when is "nicer" better than "correctness" ?!

For all you can argue about correctness, they're right in two regards:
1. The other `KEY` enums still use the numbers themselves, you should either change all of them or keep them as they are
2. This is out of the scope of this PR as currently described - the entire description of your PR is fixing an issue, yet it also edits existing enums to be defined differently.

The obvious thing to do is to change **all** of the enums to use `KEY_<X>` in their definition **in a separate PR**.

Reply:
I could bet money they'll never change. If you do change them, you'll also need to make sure they exist in server-side

Reply:
I plan to make a generator for the whole file, to fix this madness.
Therefore, closing this PR.